### PR TITLE
Remove winapi dependency

### PR DESCRIPTION
**Pull request by Ilia-Kosenkov**
Following #107  and #108.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ The output folder for bindings can be configured using `LIBRSYS_BINDINGS_OUTPUT_
 
   Add the following to the `PATH` (using `PowerShell` syntax). 
   ```pwsh
-  $env:PATH = "${env:R_HOME}\bin\x64;C:\rtools42\usr\bin;C:\rtools42\x86_64-w64-mingw32.static.posix\bin:${env:MSYS_ROOT}\mingw64\bin"
+  $env:PATH += ";${env:R_HOME}\bin\x64;C:\rtools42\usr\bin;C:\rtools42\x86_64-w64-mingw32.static.posix\bin;${env:MSYS_ROOT}\mingw64\bin"
```
Comment (yutannihilation) [inline README.md:150]: Thanks for catching! This was my mistake when I was changed this from prepending to appending in https://github.com/extendr/libR-sys/pull/94...

```suggestion
  $env:PATH = ";${env:R_HOME}\bin\x64;C:\rtools42\usr\bin;C:\rtools42\x86_64-w64-mingw32.static.posix\bin;${env:MSYS_ROOT}\mingw64\bin;${env:PATH}"
```

Comment (Ilia-Kosenkov) [reply README.md]: I actually used our readme to set up the environment to run tests and it failed, then I realized we had typos. Just FYI, in a typical installation scenario (several R versions, Rtools 4.0 and 4.2), it worked fine when appended.

Comment (yutannihilation) [reply README.md]: Yeah, it usually works, but sometimes doesn't. I forgot why I switched to prepend, but I guess that was to match with the way we do on the CI (appending to the `GITHUB_PATH` file means prepending to `PATH`, iiuc) and the CI was the case when things don't work well. Anyway, I'm open to suggestions. If you think it's better to revert this, feel free to file an issue!

**Review by yutannihilation — APPROVED** [review]
Looks good to me.

**Pull request merged**